pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -589,11 +589,16 @@ func init() {
 					Verbs:     sets.NewString("list", "watch", "get"),
 					Resources: sets.NewString("storageclasses"),
 				},
-				// Gluster provisioner
+				// Glusterfs provisioner
 				{
 					Verbs:     sets.NewString("get", "create", "delete"),
 					Resources: sets.NewString("services", "endpoints"),
 				},
+				// Glusterfs & Ceph provisioner
+				{
+					Verbs:     sets.NewString("get"),
+					Resources: sets.NewString("secrets"),
```

Comment:
opened https://github.com/kubernetes/kubernetes/pull/35675 to require the targeted secret to be a specific type